### PR TITLE
Updated get_all_courses in routes.py

### PR DESCRIPTION
**Pull request by pikamegan**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 def get_all_courses():
```
Comment (MarkoShiva) [inline app/routes.py:55]: We do also need a route definition. ;) 
Something like /all_courses

Comment (pikamegan) [reply app/routes.py:55]: Do you mean something like: @app.route('/all_courses') ?

Comment (MarkoShiva) [reply app/routes.py:55]: Yes with addition of the methods parameter. :) There is example in other routes. :)

Comment (pikamegan) [reply app/routes.py:55]: Okay, will do

**Comment by pikamegan** [comment]
> Do not forget that you need to import the functions or classes that you write in one file into another. :)

And I was wondering why I kept getting an error haha..

**Pull request merged**